src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -417,6 +527,110 @@ class DialogZWaveJSAddNode extends LitElement {
     }
   }
 
+  private async _scanQRCode(): Promise<void> {
+    this._unsubscribe();
+    this._status = "loading";
+    if (navigator.mediaDevices) {
+      const QrScanner = (await import("qr-scanner")).default;
+      if (!(await QrScanner.hasCamera())) {
+        this._error = "No camera found";
```

Comment:
Should the `status` be updated ?

Reply:
Don't think so, you can still enter the QR code manually